dbt_copilot_helper/commands/pipeline.py

```diff
@@ -92,6 +136,28 @@ def _get_git_remote():
     if not git_repo:
         return
 
-    domain, repo = git_repo.split("@")[1].split(":")
+    _, repo = git_repo.split("@")[1].split(":")
+
+    return re.sub(r".git$", "", repo)
+
+
+def _get_application_name():
+    app_name = None
+
+    try:
+        app_config = load_and_validate_config("bootstrap.yml", BOOTSTRAP_SCHEMA)
+        app_name = app_config["app"]
+    except (FileNotFoundError, ParserError):
+        pass
+
+    try:
+        if app_name is None:
+            app_config = yaml.safe_load(Path("copilot/.workspace").read_text())
+            app_name = app_config["application"]
+    except (FileNotFoundError, ParserError):
+        pass
+
+    if app_name is None:
+        abort_with_error("No valid bootstrap.yml or copilot/.workspace file found")
```

Comment:
😍 
I think it would be good to extract this to something under `utils` and make mention of it on https://uktrade.atlassian.net/browse/DBTP-468

Or just mention it on the ticket and handle the move under that.

Reply:
Done in [latest commit](https://github.com/uktrade/copilot-tools/pull/216/commits/638675261739d03737f6d83b13d44d4081a3cd05).